lab_1_classify_by_unigrams/main.py

```diff
@@ -5,6 +5,18 @@
 
 
 def tokenize(text: str) -> list[str] | None:
+    if isinstance(text, str):
```

Comment:
Try to use early return. In this case you do not need `esle` statement at all and the function is more readable

Reply:
Done